modules/lvms-installing-logical-volume-manager-operator-using-openshift-web-console.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_content-type: PROCEDURE
+[id="lvms-installing-lvms-with-web-console_{context}"]
+= Installing {lvms} using {product-title} Web Console
+
+You can install {lvms} using the Red Hat {product-title} OperatorHub.
+
+.Prerequisites
+
+* You have access to the {sno} cluster using an account with `cluster-admin` and Operator installation permissions.
```

Comment:
Can this be a 2nd bullet "You are using an account with the `clsuter-admin` and Operator installation permissions"